css/src/components/tabs.scss

```diff
@@ -0,0 +1,115 @@
+$tabs-padding: $font-size-9 $font-size-5 !default;
+$tabs-padding-lg: $font-size-7 $font-size-4 !default;
+
+$tab-font-color: $primary !default;
+$tab-border: $border-width solid transparent !default;
+$tabs-border: $border-width solid $border !default;
+$tabs-hover-background-color: $primary-background !default;
+$tabs-hover-border: $border-width solid $border !default;
+$tabs-hover-border-color: $primary-hover !default;
+$tabs-active-hover-background-color: $primary-hover !default;
+$tabs-active-hover-border-color: $primary-hover !default;
+$tabs-active-background-color: $primary-active;
+$tabs-active-border-color: $primary-active !default;
+$tabs-active-color: $text-invert !default;
+
+.tabs {
+	display: grid;
+	justify-content: center;
+	width: 100%;
+	grid-gap: $font-size-7;
+	grid-template-columns: max-content auto max-content;
+	white-space: nowrap;
+}
+
+.tab-previous,
+.tab-next {
+	display: flex;
+	align-items: center;
+	padding: $tabs-padding;
+	border: $tab-border;
+	border-radius: $border-radius-rounded;
+	background-color: $tabs-active-background-color;
+	color: $tabs-active-color;
+	font-weight: $weight-semibold;
+	box-shadow: $box-shadow-medium;
+
+	&:hover {
+		border-color: $tabs-active-hover-border-color;
+		background-color: $tabs-active-hover-background-color;
+
+		@include forced-colors {
+			border-color: Highlight;
+			background-color: Highlight;
+			color: HighlightText;
+			forced-color-adjust: preserve-parent-color;
+		}
+	}
+}
+
+.tab-list {
+	grid-column: 2;
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+	border-radius: $border-radius-rounded;
+	box-shadow: $box-shadow-medium;
+	overflow-x: hidden;
+}
+
+.tab-control {
+	display: flex;
+	position: relative;
+	align-items: center;
+	justify-content: center;
+	padding: $tabs-padding;
+	border: $tab-border;
+	border-radius: $border-radius-rounded;
+	background-color: transparent;
+	color: $tab-font-color;
+	font-weight: $weight-semibold;
+	cursor: pointer;
+	appearance: none;
+	vertical-align: top;
+
+	&:hover {
+		border-color: $tabs-hover-background-color;
+		background-color: $tabs-hover-background-color;
+		z-index: $zindex-hover;
+
+		@include forced-colors {
+			border-color: Highlight;
+			background-color: Highlight;
+			color: HighlightText;
+			forced-color-adjust: preserve-parent-color;
+		}
+	}
+
+	&[aria-selected='true'] {
+		border-color: $tabs-active-border-color;
+		background-color: $tabs-active-background-color;
+		color: $tabs-active-color;
+		z-index: $zindex-active;
+
+		&:hover {
+			border-color: $tabs-active-hover-border-color;
+			background-color: $tabs-active-hover-background-color;
+			z-index: $zindex-hover;
+		}
+
+		@include forced-colors {
+			border-color: Highlight;
+			background-color: Highlight;
+			color: HighlightText;
+			forced-color-adjust: preserve-parent-color;
+		}
+	}
+}
+
+.tabs.tabs-lg {
```

Comment:
I think we may be better off following the font pattern for sizing. 

Checkbox is an example - parent sets font sizes, inner elements can use .em for easy sizing. 
https://github.com/microsoft/atlas-design/blob/95d2e61785db71f8f113888d785162027ce6b6aa/css/src/components/form/checkbox.scss#L60

Reply:
updated

Reply:
Closing, as more recent conversations have tackled sizing.